gcp_io/gcp_interface.py

```diff
@@ -1,9 +1,8 @@
+from ast import Return
```

Comment:
what is this??

Reply:
maybe some automatic vscode import, I will remove it.